platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/async/ServiceLifecycle.java

```diff
@@ -35,21 +34,19 @@ private enum State {RUNNING, STOPPING, STOPPED}
     private final String displayName;
     private final Lock lock = new ReentrantLock();
     private final Condition condition = lock.newCondition();
+    private final Map<Thread, Integer> usages = new HashMap<>();
     private State state = State.RUNNING;
-    private Map<Thread, Integer> usages = new HashMap<Thread, Integer>();
 
     public ServiceLifecycle(String displayName) {
         this.displayName = displayName;
     }
 
     public void use(Runnable runnable) {
-        use(Factories.toFactory(runnable));
-    }
-
-    public <T> T use(Factory<T> factory) {
```

Comment:
💭 Again, this is an old type, and since nobody used the `Factory`-style method, it should be fine to remove it.